lib/chewy.rb

```diff
@@ -14,19 +15,19 @@
 require 'active_support/core_ext/object/inclusion'
 require 'active_support/core_ext/string/inflections'
 
-require 'i18n/core_ext/hash'
-require 'chewy/backports/deep_dup' unless Object.respond_to?(:deep_dup)
-require 'singleton'
-require 'base64'
-
-require 'elasticsearch'
-
 def try_require(path)
   require path
 rescue LoadError
   nil
 end
 
+try_require('i18n/core_ext/hash')
```

Comment:
This appears to have been added in order to backport deep_symbolize_keys from activesupport 4.  Since chewy depends on activesupport >= 5.2 is there any reason to keep this?

Reply:
@tindron Should I remove it and see what happens to CI?

Reply:
Go for it!